tests/unit/backend/data/test_LocalDataService.py

```diff
@@ -12,15 +13,20 @@
 from random import randint, shuffle
 from typing import List
 
+import h5py
 import pytest
 from mantid.api import ITableWorkspace, MatrixWorkspace
 from mantid.dataobjects import MaskWorkspace
+from mantid.kernel import amend_config
 from mantid.simpleapi import (
     CloneWorkspace,
+    CompareWorkspaces,
```

Comment:
Instead of `CompareWorkspaces`, use either
``` python
from mantid.testing import assert_almost_equal as assert_wksp_almost_equal
...
assert_wksp_almost_equal(ws1, ws2)
```
or 
``` python
from util.helpers import workspacesEqual
...
assert workspacesEqual(ws1, ws2)
``

Reply:
Somewhat ironically, _neither_ of those options allows me to perform the comparison that I actually require here, which is a _dead_ _equal_ comparison between the workspaces.